lua/lvim/plugins.lua

```diff
@@ -1,38 +1,38 @@
-  cmp_luasnip = "16832bb50e760223a403ffa3042859845dd9ef9d",
+  cmp_buffer = "a0fe52489ff6e235d62407f8fa72aef80222040a",
+  cmp_luasnip = "75bf6434f175206cd219f9d2bbcae154a009346c",
   cmp_nvim_lsp = "134117299ff9e34adde30a735cd8ca9cf8f3db81",
   cmp_nvim_lua = "d276254e7198ab7d00f117e88e223b4bd8c02d21",
   cmp_path = "d83839ae510d18530c6d36b662a9e806d4dceb73",
-  comment = "a6e1c127fa7f19ec4e3edbffab1aacb2852b6db3",
+  comment = "58d489fb7f18c3652adf7e8e1fff9d3281a8fc6a",
   dapinstall = "dd09e9dd3a6e29f02ac171515b8a089fb82bb425",
   fixcursorhold = "0e4e22d21975da60b0fd2d302285b3b603f9f71e",
-  friendly_snippets = "e3506d575e120253b4aa47ec2100786fd1e9c38d",
-  gitsigns = "119ebde2de917399aa89ad3664000a0e1750895a",
-  lualine = "1ae4f0aa74f0b34222c5ef3281b34602a76b2b00",
-  luasnip = "4191d3e475b65407b8ccf47e4dcbf38552fdbb2d",
-  nlsp_settings = "199bc1b2859206488423b676da4a7d010b1a3373",
-  null_ls = "73420db2b58408a60a19024be3c14b0eba8413fe",
-  nvim_autopairs = "18fe311bb967d16ddf2cc28e7e71f234c37d3e26",
-  nvim_cmp = "d12ba90da372bfe5d9a103546ce1553341a2daff",
-  nvim_dap = "ce4e56f76598881d020e68037ae91310ea8b9d54",
-  nvim_lsp_installer = "4df84d7d3e8bcc9b98a1436b6b4155b3a727d41d",
-  nvim_lspconfig = "382712c4ecba5816f398379dd9d8c8200b025ca3",
-  nvim_notify = "3f7af5878a890503f9efb7de9f2a0b3f895ad518",
-  nvim_tree = "e842f088847c98da59e14eb543bde11c45c87ef7",
-  nvim_treesitter = "d6a0a26b8563409d4660def7320a4f4bc23954df",
+  friendly_snippets = "4a9516c116f8d3a5766fcb8ac91b176979612d5d",
+  gitsigns = "aaf680472187798d5945e39179b540bd3bf80341",
+  lualine = "5596c2a25a49ca235613c804169b9063e20b05f5",
+  luasnip = "577045e9adf325e58f690f4d4b4a293f3dcec1b3",
+  nlsp_settings = "599edc32707f53bd9b0739879f013b3bf162ea4e",
+  null_ls = "fb9e2a64ae8e43c2255025064cfee37dc7d6a752",
+  nvim_autopairs = "04cd1779f81e9d50d5a116c5dccd054b275bd191",
+  nvim_cmp = "af07ff9b7973e95eff9e0275e13fe0350281208b",
+  nvim_dap = "3b3027e0ca98775000e1ba727d8f292e821f9f03",
+  nvim_lsp_installer = "fcd5d79a7f4966645e9d2f512b85b72f7de0dde2",
+  nvim_lspconfig = "e6d95863a336b7e52c92b38c62aa60b469254d14",
+  nvim_notify = "ef027e34b618eac42fb0111c1db670ba01793039",
+  nvim_tree = "2e33b1654384921ec1cc9656a2018744f3f1ce81",
+  nvim_treesitter = "1d66657e6d0f1f8f79ddc48ff1dac9788694cc2d",
   nvim_ts_context_commentstring = "9f5e422e1030e7073e593ad32c5354aa0bcb0176",
-  nvim_web_devicons = "8df4988ecf8599fc1f8f387bbf2eae790e4c5ffb",
+  nvim_web_devicons = "344331467509802e1af200f08ec3da278be5cbba",
   packer = "851c62c5ecd3b5adc91665feda8f977e104162a5",
-  plenary = "1c31adb35fcebe921f65e5c6ff6d5481fa5fa5ac",
+  plenary = "e6267f79481064eee53950571f53cbaafb08417d",
   popup = "b7404d35d5d3548a82149238289fa71f7f6de4ac",
   project = "71d0e23dcfc43cfd6bb2a97dc5a7de1ab47a6538",
   structlog = "6f1403a192791ff1fa7ac845a73de9e860f781f1",
-  telescope = "27294d73e4562cbc83c9c488b4618f012deade7e",
+  telescope = "5f37fbfa837dfee7ecd30f388b271f4a71c0a9e0",
```

Comment:
```suggestion
  telescope = "80cdb00b221f69348afc4fb4b701f51eb8dd3120", -- see telescope.nvim#1549
```

We need to make a decision about telescope since they have made 0.6 a requirement, but that should probably be in a different PR since it's going to require some extra work updating all the bindings.

![image](https://user-images.githubusercontent.com/59826753/145669487-57015df7-c0b9-41b9-9dde-1ae223a7cd0d.png)